lib/matplotlib/figure.py

```diff
@@ -1552,6 +1552,9 @@ def subfigures(self, nrows=1, ncols=1, squeeze=True,
         the same as a figure, but cannot print itself.
         See :doc:`/gallery/subplots_axes_and_figures/subfigures`.
 
+        .. note::
+            ``subfigure`` is new in v3.4, and the API is still provisional.
```

Comment:
Should this be `subfigures`? (As in the method name?)

Reply:
What is this?

Reply:
I've just copied from https://matplotlib.org/stable/gallery/subplots_axes_and_figures/subfigures.html#figure-subfigures

Likely it's intended to be broader than just the single method, e.g.

> The *subfigure* concept is new in v3.4, ...

Possibly we should rewrite both notes that way, and also add it to https://matplotlib.org/stable/api/figure_api.html#matplotlib.figure.SubFigure I will follow up with a PR later.